crates/services/p2p/src/discovery.rs

```diff
@@ -369,7 +368,7 @@ mod tests {
                                     .add_address(&peer_id, unroutable_peer_addr.clone());
                             }
                             SwarmEvent::ConnectionClosed { peer_id, .. } => {
-                                panic!("PeerId {peer_id:?} disconnected");
+                                dbg!(peer_id);
```

Comment:
Done

Reply:
It seems to make the test timeout : https://github.com/FuelLabs/fuel-core/actions/runs/11813227788/job/32909924335#step:8:2439

Should I remove this ?

Reply:
Hmm, I think we should rewrite `discovery_works` test. And maybe we should use the service itself instead of the p2p service. You can leave the previous implementation and create a new tech debt to rewrite this test.

Reply:
Done https://github.com/FuelLabs/fuel-core/issues/2434